### ISSUE TITLE
[Bug]: Desktop client 3.13.2-x64 crashing on Windows 10 Pro 19045.4651 with null_pointer on cldapi.dll 

### ISSUE THREAD

**Issue by nthoms92** [nextcloud/desktop#6918]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Fresh install of NextCloud server/client.  The client has never run successfully on this PC.    The client will run for 30 seconds to a maximum of 2 minutes per my testing until it ultimately crashes.  Windbg shows the below 

MODULE_NAME: cldapi
IMAGE_NAME:  cldapi.dll
FAILURE_BUCKET_ID:  NULL_POINTER_WRITE_c0000005_cldapi.dll!IssueHsmControl

I have confirmed the PC doesn't have any other kind of Sync software running, and I have excluded the NextCloud client software/directory from my antivirus. I have attempted a reinstall of the software as well with no effect.  

Prior to the crash, I do see it marking files for synchronization per expected operation.  

### Steps to reproduce

Install NextCloud client software  
Run the software. 
Wait for a few minutes 
Software crashes.  

### Expected behavior

Expected software to remain running/stable after launch 

### Which files are affected by this bug

cldapi.dll, nextcloud.exe

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10 Pro 19045.4651 

### Package

Appimage

### Nextcloud Server version

29.0.3

### Nextcloud Desktop Client version

3.13.2-x64

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### Are you using an external user-backend?

- [x] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
attached to zip in additional info.
```

### Additional info

[Next Cloud Crash.zip](https://github.com/user-attachments/files/16228297/Next.Cloud.Crash.zip)

I am unable to use any GUI features of the NextCloud client, and setting the debug logging through the cfg file results in explorer.exe freezing, a "blank" error popup, and no log written.

**Comment by nthoms92** [comment]
I figured it out.  This was caused by a problem in the nextcloud files in %appdata%/local and/or %appdata%/roaming.  Neither of which are removed on uninstall of the NextCloud application (that should be a bug).  Due to something in one of those directories I was having this issue.  I uninstalled the app, deleted both offending directories which once again are not removed by the client uninstall, then reinstalled.  Its now working normally
